org-formation/organization.yaml

```diff
@@ -37,24 +44,29 @@ Organization:
         - !Ref SynapseOU
         - !Ref BridgeOU
 
-  ITOU:
+  ItDevOU:
+    Type: OC::ORG::OrganizationalUnit
+    Properties:
```

Comment:
This definition doesn't need an `OrganizationalUnits` property?

Reply:
Nope, it can be a standalone OU with nothing in it.